sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseBulkEditData.java

```diff
@@ -70,6 +71,7 @@
 	private FacilityType facilityType;
 	private FacilityReferenceDto healthFacility;
 	private String healthFacilityDetails;
+	private Boolean dontShareWithReportingTool;
```

Comment:
Why not make it primitive? This would be cheaper.

Reply:
Because it should be null when no change wanted for this field